Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -217,17 +219,25 @@ class BackForwardListViewController: UIViewController, UITableViewDataSource, UI
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-       let cell = self.tableView.dequeueReusableCell(withIdentifier: BackForwardListCellIdentifier, for: indexPath) as! BackForwardTableViewCell
+        let cell = self.tableView.dequeueReusableCell(withIdentifier: BackForwardListCellIdentifier, for: indexPath) as! BackForwardTableViewCell
         let item = listData[indexPath.item]
         let urlString = item.url.isLocal ? item.url.getQuery()["url"]?.unescape() : item.url.absoluteString
-        guard let url = urlString else {
-            return cell // This should never happen.
+        
+        // Mozilla Firefox Start Page
+        if urlString == nil || urlString?.range(of: "http://localhost:6571/") != nil  {
```

Comment:
I would be careful about hard coding the port we use for local web pages. If we want to change the port later one, it will break this feature and the compiler won't tell us much. Along with the suggestion above, we should try to only whitelist the home page and leave the other about pages not visible.